src/foam/nanos/rope/ROPEUtils.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.rope',
+  name: 'NextRelationshipsList',
```

Comment:
Do we need this model? Doesn't seem useful to have a model with one array property. Couldn't we just use a List<String> directly wherever we use this? It doesn't seem to add any other useful information to the list.

Reply:
the list in the map gets interpreted as an array, I think, which is why it was showing up weird before when reading from journal as something like <string, Ljava.lang.string@>, instead of <string, list>.